Classes/Backend/DynamicFlexForm.php

```diff
@@ -323,7 +320,10 @@ protected function recursivelyEvaluateClosures(array $dataStructureArray, array
      */
     protected function getCache()
     {
-        return GeneralUtility::makeInstance(CacheManager::class)->getCache('flux');
+        if ($this->cache !== null) {
```

Comment:
It does, but we save a bunch of calls in GeneralUtility as well. Your choice - I'd be happy to remove it if you think it's better without a local cache.